docs/UserGuide.md

```diff
@@ -1,134 +1,162 @@
-
 * [Quick Start](#quick-start)
 * [Features](#features)
 * [FAQ](#faq)
 * [Command Summary](#command-summary)
 
 ## Quick Start
 
-0. Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
+Step 1: Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
    > Having any Java 8 version is not enough. <br>
-   This app will not work with earlier versions of Java 8.
+   This application will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds. 
-   > <img src="images/Ui.png" width="600">
-
-4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
-5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
+Step 2: Download the latest `taskell.jar` from here.<br>
+	<br> <img src="images/Icon.png" width="100"></br>
+Step 3: Copy the file to the folder you want to use as the home folder for your Task Manager.<br>
+Step 4: Double-click the file to start the application. The GUI should appear in a few seconds. <br>
+ <br><img src="images/GUI.png" width="600"><br>
+Picture 1: A screenshot of the Graphical User Interface (GUI)<br>
+<br>Step 5:	Type the relevant command in the command box and press <kbd>Enter</kbd> to execute it.<br>
+Step 6: Some example commands you can try:<br>
+   * **`list`** : `lists` all contacts
+   * **`add`**` buy MA1101R textbook today` : adds a task called buy MA1101R textbook to be done by today.
+   * **`delete`**` 3` : deletes the 3rd task shown in the current list
+   * **`exit`** : exits the application <br>
+Refer to the [Features](#features) section below for details of each command.<br>
 
 
 ## Features
 
-> **Command Format**
+**Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The parameters have to be in the given order below.
 
-#### Viewing help : `help`
-Format: `help`
+#### Viewing list of commands : `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+When unsure of the available commands or command format, <kbd>Enter</kbd> help. 
+If you enter an incorrect command, eg. search, the instruction for using each command will be displayed as well.<br>
```

Comment:
in the normal sense `search` is a perfectly valid command. try to use a better gibberish for incorrect command :-)